test/terraorg/model/org_test.rb

```diff
@@ -53,18 +47,286 @@ def test_org_tf_generation
+        return Org.new(JSON.parse(org_data), platoons, squads, people, gsuite_domain)
+    end
+
+    $WORKING_ORG =  <<-JSON
+            {
+        "exception_people": [
+            {
+            "location": "US",
+            "members": [
+                "exception1"
+            ]
+            }
+        ],
+        "exception_squads": [],
+        "id": "contoso",
+        "manager": "rchen",
+        "manager_location": "US",
+        "metadata": {},
+        "name": "Contoso",
+        "platoons": [
+            "sprockets",
+            "widgets"
+        ],
+        "version": "v1"
+        }
+    JSON
+
+    $WORKING_PLATOONS =  <<-JSON
+    {
+        "platoons": [
+          {
+            "id": "sprockets",
+            "manager": "manager2",
+            "name": "Sprockets",
+            "squads": [
+              "legal",
+              "logistics"
+            ]
+          },
+          {
+            "id": "widgets",
+            "manager": "manager1",
+            "name": "Widgets",
+            "squads": [
+              "marketing",
+              "sales"
+            ]
+          }
+        ],
+        "version": "v1"
+      }
+      
+    JSON
+
+    $WORKING_SQUADS =  <<-JSON
+    {
+  "squads": [
+    {
+      "id": "legal",
+      "metadata": {
+        "epo": "epo4",
+        "manager": "manager5",
+        "pm": [
+          "pm1"
+        ],
+        "slack": "#legal",
+        "sme": "sme8"
+      },
+      "name": "Legal",
+      "team": [
+        {
+          "location": "US",
+          "members": [
+            "member1",
+            "member2"
+          ]
+        }
+      ]
+    },
+    {
+      "id": "logistics",
+      "metadata": {
+        "epo": "epo9",
+        "manager": "manager11",
+        "pm": [
+          "pm9"
+        ],
+        "slack": "#logistics",
+        "sme": "sme9"
+      },
+      "name": "Logistics",
+      "team": [
+        {
+          "location": "CN",
+          "members": [
+            "member55",
+            "member56"
+          ]
+        }
+      ]
+    },
+    {
+      "id": "marketing",
+      "metadata": {
+        "manager": "manager2",
+        "pm": [
+          "pm2"
+        ],
+        "slack": "#marketing",
+        "sme": "sme2"
+      },
+      "name": "Marketing",
+      "team": [
+        {
+          "location": "FR",
+          "members": [
+            "member3",
+            "member4"
+          ]
+        }
+      ]
+    },
+    {
+      "id": "sales",
+      "metadata": {
+        "epo": "epo1",
+        "manager": "epo2",
+        "pm": [
+          "pm1"
+        ],
+        "slack": "#sales",
+        "sme": "sme1"
+      },
+      "name": "Sales",
+      "team": [
+        {
+          "location": "GB",
+          "members": [
+            "member8",
+            "member9"
+          ]
+        }
+      ]
+    }
+  ],
+  "version": "v1"
+}
+
+    JSON
+
+    $ORPHANED_ASSOCIATE_SQUADS =  <<-JSON
```

Comment:
nit, not required to merge: would it be better to make these fixture files that are read from disk?

Reply:
Yeah,  that was how I had it before, but switched to this pattern to make it a little easier to make changes. I don't feel strongly about it.